nba-reference/src/components/stats-table.tsx

```
@@ -165,61 +137,84 @@ export function StatsTable({ columns, rows, initialSort }: StatsTableProps) {
+                          // Toggle direction if clicking same column
+                          setDirection(currentDirection =>
+                            currentDirection === 'asc' ? 'desc' : 'asc'
+                          );
+                        } else {
+                          // New column: default to descending
+                          setSortKey(column.key);
+                          setDirection('desc');
+                        }
+                      }}
+                      className={tableHeaderButtonClass}
+                    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)